app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -69,24 +81,22 @@ export default {
       },
     };
 
-    useKeyboardEvents(keyboardEvents);
-
-    const {
-      isAIIntegrationEnabled,
-      isLabelSuggestionFeatureEnabled,
-      fetchIntegrationsIfRequired,
-      fetchLabelSuggestions,
-    } = useAI();
+    useKeyboardEvents(keyboardEvents, conversationFooterRef);
```

Comment:
We don't need this ref usage in the `useKeyboardEvents` composable.
```suggestion
    useKeyboardEvents(keyboardEvents);
```